src/components/SettingsPage/sections/CryptoAssets/RateRow/PriceGraph.js

```diff
@@ -8,15 +8,17 @@ import CounterValues from 'helpers/countervalues'
 import { PlaceholderLine } from 'components/Placeholder'
 import { colors } from 'styles/theme'
 import { rgba } from 'styles/helpers'
+import { getDates } from '@ledgerhq/live-common/lib/portfolio'
+import { timeRangeDaysByKey } from '../../../../../reducers/settings'
 
-const DAY = 24 * 60 * 60 * 1000
 const mapStateToProps = (state, props: *) => {
+  const dates = getDates(props.timeRange)
+  const days = timeRangeDaysByKey[props.timeRange]
```

Comment:
```suggestion
  const days = dates.length
```

isn't it the same? (in fact it shouldn't be named days, because might not be days in the future)
you can probably drop this param and just uses `dates.length` when it was used (only once for isAvailable?)

Reply:
Indeed